src/index.js

```diff
@@ -0,0 +1,94 @@
+import 'lodash';
+import './style.css';
+import elementChanges from './check.js';
+
+let ToDoArray = JSON.parse(localStorage.getItem('ToDoArray') || '[]');
+let title = JSON.parse(localStorage.getItem('title'));
+if (!ToDoArray.length) {
+  document.querySelector('.Activity-list').classList.add('hidden');
+} else {
+  document.querySelector('.Activity-list').classList.remove('hidden');
+}
+document.querySelector('#whatToDo').value = title;
+
+function CreateOb({ description, index }) {
+  this.description = description;
+  this.completed = false;
+  this.index = index;
+}
+
+function displayOnScreen(element) {
+  const ul = document.querySelector('.form_');
+  const li = document.createElement('li');
+  li.classList.add('toDo');
+  const checkbox = document.createElement('input');
+  const br = document.createElement('br');
+  const image = document.createElement('img');
+  image.setAttribute('src', 'icons/delete.PNG');
```

Comment:
- [ ] As requested by the previous reviewer, the link to your delete icon is broken, please kindly fix that. See image below.
![deleteBtn](https://user-images.githubusercontent.com/65852621/144520053-226cab73-1660-48f7-93bc-cee4f54b0d2c.png)